cirq/ops/eigen_gate.py

```diff
@@ -98,17 +99,20 @@ def __init__(self, *,  # Forces keyword args.
                 `cirq.Rx(t)` uses a `global_shift` of -0.5, which is why
                 `cirq.unitary(cirq.Rx(pi))` equals -iX instead of X.
         """
-        self._exponent = exponent
+        if isinstance(exponent, sympy.Basic):
```

Comment:
Unconditionally assign exponent. We're happy with whatever we were given. Definitely don't try to pick a randomvariable out of an equation as the exponent.

Reply:
Done.